octoprint_octorelay/driver.py

```diff
@@ -40,8 +37,29 @@ def toggle(self, desired_state: Optional[bool] = None) -> bool:
         """
         if desired_state is None:
             desired_state = not self.is_closed()
-        GPIO.setwarnings(False)
-        GPIO.setup(self.pin, GPIO.OUT)
-        GPIO.output(self.pin, xor(self.inverted, desired_state))
-        GPIO.setwarnings(True)
+        
+        if self.__xor(self.inverted, desired_state) is True:
+            self.relay.on()
+        else:
+            self.relay.off()
+
         return desired_state
+
+    @classmethod
+    def get_or_create_relay(cls, pin: int, inverted: bool):
+        to_return = None
+
+        for relay in cls.relays:
+            if relay.pin == pin:
+                if relay.inverted != inverted:
```

Comment:
```suggestion
                if relay.inverted is not inverted:
```

I heard it's better to use this syntax for comparing booleans